src/Components/Publishing/Header/Layouts/ClassicHeader.tsx

```diff
@@ -51,14 +51,6 @@ export const Title = styled.div`
   padding-bottom: ${space(3)}px;
   ${garamond("s37")};
 
-  h1 {
-    font-style: inherit;
-    font-family: inherit;
-    font-weight: inherit;
-    font-size: inherit;
-    margin: 0;
-  }
-
```

Comment:
@eessex - I removed these following the other PR.

Reply:
Cool, thanks for taking this off my todo's